cmd/poller/poller.go

```diff
@@ -575,6 +576,39 @@ func (me *Poller) loadCollector(class, object string) error {
 	return nil
 }
 
+func Union2(template *node.Node, params *conf.Poller) {
```

Comment:
Could you add some examples or comments in this method. Not very clear to follow.